dockerfiles/Dockerfile.dev

```diff
@@ -3,8 +3,11 @@ FROM    golang:1.8.3-alpine
 
 RUN     apk add -U git make bash coreutils
 
-RUN     go get github.com/LK4D4/vndr && \
-        cp /go/bin/vndr /usr/bin && \
+ARG     VNDR_COMMIT=9909bb2b8a0b7ea464527b376dc50389c90df587
```

Comment:
should go in `hack/dockerfile/binaries-commits`

`hack/dockerfile/install-binaries.sh` will override this version otherwise